api/src/flows.ts

```diff
@@ -421,9 +421,11 @@ class FlowManager {
 
 		const handler = this.operations.get(operation.type)!;
 
-		const options = applyOptionsData(operation.options, keyedData);
+		let options = null;
```

Comment:
Defaulting to the options value would be helpful for debugging via logs.

```suggestion
		let options = operation.options;
```

Reply:
Agreed and good enough for me